instrumentation/instaawssdk/instaawssdk.go

```diff
@@ -96,5 +108,44 @@ func injectTraceContext(sp opentracing.Span, req *request.Request) {
 			opentracing.TextMap,
 			SNSMessageAttributesCarrier(params.MessageAttributes),
 		)
+	case *lambda.InvokeInput:
+		lc := LambdaClientContext{}
+
+		if params.ClientContext != nil {
+			res, err := base64.StdEncoding.DecodeString(*params.ClientContext)
+			if err != nil {
+				sp.LogFields(otlog.Error(req.Error))
```

Comment:
This would mark the span potentially started by user as erroneous, which may trigger an alarm signalling there is something wrong with the code. I'd suggest we simply log this instead.

Reply:
valid point